CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
This field appears to be a temporary or debug variable and is not appropriate in a production domain model. Leaving TODOs and unused fields in domain entities violates clean architecture and can lead to confusion or misuse.

```suggestion
// Removed temporary salary field before merging
```